packages/@react-spectrum/combobox/src/ComboBox.tsx

```diff
@@ -48,15 +49,16 @@ function ComboBox<T extends object>(props: SpectrumComboBoxProps<T>, ref: RefObj
     menuTrigger = 'input',
     autoFocus,
     shouldFlip = true,
-    width,
     direction = 'bottom'
   } = props;
 
+  let {hoverProps, isHovered} = useHover(props);
   let {styleProps} = useStyleProps(props);
   let popoverRef = useRef<DOMRefValue<HTMLDivElement>>();
   let triggerRef = useRef<FocusableRefValue<HTMLElement>>();
   let listboxRef = useRef();
   let inputRef = useRef<HTMLInputElement & HTMLTextAreaElement>();
+  let {pressProps, isPressed} = usePress({});
```

Comment:
Bleh, feels bad to do all this stuff just to get the active state on the trigger button on mousedown of the label, lemme know if there is a more elegant way of handling

Reply:
Nvm, got rid of this entirely since usePress's passing through of all other props as `domProps` complicates things